docs/querying/aggregations.md

```diff
@@ -27,10 +27,10 @@ title: "Aggregations"
 > language. For information about aggregators available in SQL, refer to the
 > [SQL documentation](sql.md#aggregation-functions).
 
-Aggregations can be provided at ingestion time as part of the ingestion spec as a way of summarizing data before it enters Apache Druid.
-Aggregations can also be specified as part of many queries at query time.
+Aggregations specified at ingestion time, from within the ingestion spec, provide a way to summarize data before it enters Apache Druid.
+Aggregations are also used at query time.
 
-Available aggregations are:
+The following sections list the types of aggregations available at ingestion and query time.
```

Comment:
```suggestion
The following sections list the available aggregate functions. Unless otherwise noted aggregations are available at both at ingestion and query time.
```
i think they are "aggregate functions" but may be wrong here.